### DISCUSSION TITLE
what's different between minio_cluster_capacity_raw_free_bytes and minio_cluster_capacity_usable_free_bytes?

### DISCUSSION THREAD

**Discussion by lddlww** [minio/minio#14306] [Q&A] [answered]
what's different with ```minio_cluster_capacity_raw_free_bytes``` and ```minio_cluster_capacity_usable_free_bytes```

**Comment by klauspost** [comment, marked as answer]
`minio_cluster_capacity_raw_free_bytes` does not consider parity. `minio_cluster_capacity_usable_free_bytes` is the free capacity when using the standard parity for your cluster.

**Reply by lddlww** [reply]
thank you very much
